options.proto

```diff
@@ -90,6 +90,8 @@ extend google.protobuf.FileOptions {
 
 extend google.protobuf.MessageOptions {
   MessageOptions message_options = 1127;
+
+  string message_manual_link = 50000;
```

Comment:
I was unable to name this `manual_link` because of the option in `FieldOptions`
(was a little confused because I thought fields could be duplicated)